analyze.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import argparse
+from pathlib import Path
+import sys
+
+# path to all benchmark modules
+sys.path.append("analyze/")
+if sys.version_info[0] < 3:
+    raise "Python 3 should be used to run this script"
```

Comment:
Would it be possible to rewrite the script so it works both on Python 2 and 3? If not, then I agree, it's better to aim for v3.

Reply:
If you depend on python 3, please make the initial line in the script state that:
    #!/usr/bin/env python3

Reply:
@jbech-linaro you're right, it's better to do ASAP until there is no much code here

Reply:
@jbech-linaro decided to check if there are any python2 incompatibilities, found out that after installing back-ported `enum34` and `pathlib` everything works fine without any changes :)